cluster.yaml

```diff
@@ -13,7 +13,6 @@ spec:
     value:
       apiVersion: baremetalproviderspec/v1alpha1
       kind: BareMetalClusterProviderSpec
-      sshKeyPath: cluster-key
```

Comment:
It this not necessary any more?

Reply:
not any more. wksctl removed this config from 0.8.x.